compiler+runtime/src/cpp/jank/runtime/obj/cons.cpp

```diff
@@ -102,6 +102,11 @@ namespace jank::runtime
     return runtime::to_string(seq());
   }
 
+  native_persistent_string obj::cons::to_code_string() const
+  {
+    return to_string();
```

Comment:
Same concern with `chunked_cons`. This is a sequence. We need to use `runtime::to_code_string(seq());`.